cl/favorites/utils.py

```diff
@@ -144,7 +156,47 @@ async def get_top_prayers() -> list[RECAPDocument]:
         .order_by("-geometric_mean")[:50]
     )
 
-    return [doc async for doc in documents.aiterator()]
+    return documents
+
+
+async def get_user_prayers(user: User) -> QuerySet[Prayer]:
+    user_prayers = Prayer.objects.filter(user=user).values("recap_document_id")
+
+    documents = (
+        RECAPDocument.objects.filter(id__in=Subquery(user_prayers))
+        .select_related(
+            "docket_entry",
+            "docket_entry__docket",
+            "docket_entry__docket__court",
+        )
+        .only(
+            "pk",
+            "document_type",
+            "document_number",
+            "attachment_number",
+            "pacer_doc_id",
+            "page_count",
+            "is_free_on_pacer",
```

Comment:
Similarly to `get_top_prayers` we should include `case_name_full` and `case_name_short`.

Additionally, some other fields are being used in the template:
```
date_upload
filepath_local
filepath_ia
```